src/gateway/get-started/load-balancing.md

```diff
@@ -1,66 +1,81 @@
 ---
 title: Configure Load Balancing
+content-type: tutorial
+book: get-started
+chapter: 6
 ---
 
-In this topic, you’ll learn about configuring upstream services, and create multiple targets for load balancing.
+Load balancing is a method of distributing API request traffic across
+multiple upstream services that can process them. Load balancing is useful for multiple reasons
 
-If you are following the getting started workflow, make sure you have completed [Secure Services Using Authentication](/gateway/{{page.kong_version}}/get-started/comprehensive/secure-services) before moving on.
+In the following example, you’ll use an application deployed across two different servers, or upstream targets. 
+{{site.base_gateway}} needs to load balance across both servers, so that if one of the servers is unavailable, 
+it automatically detects the problem and routes all traffic to the working server.
 
-## What are Upstreams?
+An [upstream](/gateway/latest/admin-api/#upstream-object) refers to the service applications sitting behind {{site.base_gateway}}, 
```

Comment:
Would this be a better link for "upstreams"? https://deploy-preview-4337--kongdocs.netlify.app/gateway/latest/understanding-kong/key-concepts/upstreams/ 

I thought they might like an explanation topic since it's a getting started topic.

Reply:
Yep, great idea.  Almost always better to link to an "explainer" than a reference

Reply:
@cloudjumpercat see latest, Thank you